src/Core/Extensible.php

```diff
@@ -511,7 +511,7 @@ public function getExtensionInstances()
         // Setup all extension instances for this instance
         $this->extension_instances = [];
         foreach (ClassInfo::ancestry(static::class) as $class) {
-            if (in_array($class, self::$unextendable_classes)) {
+            if (in_array($class, self::class::$unextendable_classes)) {
```

Comment:
Should be self::class as per https://docs.silverstripe.org/en/5/contributing/coding_conventions/php_coding_conventions/#other-conventions

Fixes issue
```
1) SilverStripe\ORM\Tests\Search\SearchContextTest::testUserDefinedFieldsAppearInSearchContext
Failed asserting that two objects are equal.
--- Expected
+++ Actual
@@ @@
             'failover' => null
             'customisedObject' => null
             'objCache' => []
-            'extension_instances' => null
+            'extension_instances' => [...]
```